cloud/grpc.go

```diff
@@ -100,26 +103,72 @@ func (c *Client) StreamInterceptor() grpc.StreamClientInterceptor {
 		ctx = c.withReqID(ctx)
 		ctx, err := c.reAuthIfExpired(ctx)
 		if err != nil {
-			return nil, errors.Wrap(err, "failed refreshing expired token")
+			return nil, errors.Wrapf(err, "failed refreshing expired token {reqID: %s}", getReqID(ctx))
 		}
 		newStreamer, err := streamer(ctx, desc, cc, method, opts...)
 		if err != nil {
 			s, ok := status.FromError(err)
 			if !ok {
-				return newStreamer, err
+				return nil, fmt.Errorf("%s {reqID: %s}", err.Error(), getReqID(ctx))
```

Comment:
why aren't you using `errors.Wrapf` like in the other error handling of this code?

Reply:
Couple reasons:
* I don't want to lose the request ID if someone prints `errors.Cause(err)` (which is exactly what we [usually do](https://github.com/earthly/earthly/blob/main/cmd/earthly/app/run.go#L252))
* I didn't have a meaningful message to add, and just wanted leave the original error unaltered while appending the request ID to it

TBH I don't think I should use `errors.Wrapf` above either

Reply:
> TBH I don't think I should use errors.Wrapf above either with the risk mention in the first point

sounds good -- we should be consistent in how we do it.